quinn-proto/src/connection/mod.rs

```diff
@@ -3127,6 +3194,12 @@ impl Connection {
         self.rem_cids.active_seq()
     }
 
+    /// Returns the detected max_udp_payload_size
+    #[cfg(test)]
+    pub(crate) fn max_udp_payload_size(&self) -> u16 {
```

Comment:
This is very easy to confuse with the transport parameter of the same name. Should we call it something like `path_mtu`, perhaps?